db/skipped_sequence.go

```diff
@@ -289,3 +301,31 @@ func (s *SkippedSequenceSlice) getOldest() uint64 {
 	// grab fist element in slice and take the start seq of that range/single sequence
 	return s.list[0].getStartSeq()
 }
+
+// getSliceLength retrieves the current skipped sequence slice length
```

Comment:
Since we only have a single caller of these functions (updateStats), I think it would make sense to define a SkippedSequenceSlice.getStats() function that returned all four of these in one lock.